brewtils/rest/client.py

```diff
@@ -18,6 +18,11 @@
 from brewtils.rest import normalize_url_prefix
 from brewtils.specification import _CONNECTION_SPEC
 
+try:
+    from urllib.parse import quote_plus
```

Comment:
The requests library has an alias for this that I believe you can use to avoid this whole try except thing:

```python
from requests.utils import quote
```

Reply:
Much easier, thank you :+1: